puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -4,6 +4,18 @@ import psycopg2.extensions
 import select
 import time
 import logging
+import six.moves.configparser
+
+REMOTE_POSTGRES_HOST=''
+REMOTE_POSTGRES_SSLMODE=''
+execfile("/etc/zulip/settings.py")
```

Comment:
Can we replace that with just `from django.conf import settings` to match how all our other tools work?

I think with that we can also remove the need for this to directly parse `zulip-secrets.conf`, which would be good, since I think this change may break the development environment and I'd like to have the path conditional for that file just live in one place in `settings.py`